testcases/response/body/missing body found when empty expected.json

```diff
@@ -0,0 +1,9 @@
+{
+  "match": true,
+  "comment": "Missing body found, when an empty body was expected",
+  "expected" : {
+    "body": null
+  },
+  "actual": {
+  }
+}
```

Comment:
@bethesque The real actual request wouldn't. You could make that argument with all the body tests in this repo (non of them have status code  specified). I assume we are doing this as the focus of the body tests is on body and not the status code.